websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,142 @@
+const presence = new Presence({
+		clientId: "968880591003082783"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/7V0kxEb.png",
+			startTimestamp: browsingTimestamp
+		},
+		path = document.location.pathname.toLowerCase();
+
+	if (path == "/") presenceData.details = "Browsing main page...";
+	else if (path == "/dashboard") presenceData.details = "Browsing dashboard...";
+	else if (path.includes("/users")) {
+		const username = document.querySelector(
+			"#avatar-wrapper > h1 > a"
```

Comment:
Seems volatile, is there a class you could select instead?

Reply:
This please

Reply:
![image](https://user-images.githubusercontent.com/63014799/173237786-6a2b4d04-7ab7-4557-b75a-2eed52e5aa4d.png)
I can't do anything with it

Reply:
instead of h1 you can use .emojis-supported

Reply:
I bet this is dynamicly added class and it won't be there if for some reason emojis won't supported or something along those lines

Reply:
okay, I changed it